### ISSUE TITLE
Error when getting player's IP in "on connect" event

### ISSUE THREAD

**Issue by TrademarkTM** [bensku/Skript#233]
Hey!
I get the following error with this code:

``
on connect:
    broadcast "%ip of player%"
``

``
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! [Skript] Severe Error:
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! If you're developing an add-on for Skript this likely means that you have done something wrong.
[11:14:44 ERROR]: #!#! If you're a server admin however please go to https://github.com/bensku/Skript/issues/
[11:14:44 ERROR]: #!#! and check whether this error has already been reported.
[11:14:44 ERROR]: #!#! If not please create a new ticket with a meaningful title, copy & paste this whole error into it (or use paste service),
[11:14:44 ERROR]: #!#! and describe what you did before it happened and/or what you think caused the error.
[11:14:44 ERROR]: #!#! If you think that it's a trigger that's causing the error please post the trigger as well.
[11:14:44 ERROR]: #!#! By following this guide fixing the error should be easy and done fast.
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! Stack trace:
[11:14:44 ERROR]: #!#! java.lang.NullPointerException
[11:14:44 ERROR]: #!#!     at ch.njol.skript.expressions.ExprIP.convert(ExprIP.java:52)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.expressions.ExprIP.convert(ExprIP.java:1)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.registrations.Converters.convert(Converters.java:305)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.registrations.Converters.convertUnsafe(Converters.java:296)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.expressions.base.PropertyExpression.get(PropertyExpression.java:105)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.expressions.base.SimplePropertyExpression.get(SimplePropertyExpression.java:56)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.expressions.base.PropertyExpression.get(PropertyExpression.java:76)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.util.SimpleExpression.getArray(SimpleExpression.java:103)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.VariableString.toString(VariableString.java:383)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.VariableString.getArray(VariableString.java:488)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.VariableString.getArray(VariableString.java:1)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.effects.EffBroadcast.execute(EffBroadcast.java:69)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:54)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:65)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:93)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:53)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:152)
[11:14:44 ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:112)
[11:14:44 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[11:14:44 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[11:14:44 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.PlayerList.attemptLogin(PlayerList.java:456)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.LoginListener.b(LoginListener.java:118)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.LoginListener.c(LoginListener.java:54)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.NetworkManager.a(NetworkManager.java:231)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.ServerConnection.c(ServerConnection.java:148)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:814)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654)
[11:14:44 ERROR]: #!#!     at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557)
[11:14:44 ERROR]: #!#!     at java.lang.Thread.run(Unknown Source)
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! Version Information:
[11:14:44 ERROR]: #!#!   Skript: 2.2-dev20
[11:14:44 ERROR]: #!#!   Bukkit: 1.8.8-R0.1-SNAPSHOT
[11:14:44 ERROR]: #!#!   Minecraft: 1.8.8
[11:14:44 ERROR]: #!#!   Java: 1.8.0_111 (Java HotSpot(TM) 64-Bit Server VM 25.111-b14)
[11:14:44 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! Running CraftBukkit: false
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! Current node: null
[11:14:44 ERROR]: #!#! Current item: broadcast "%the IP address of the player%"
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! Thread: Server thread
[11:14:44 ERROR]: #!#!
[11:14:44 ERROR]: #!#! End of Error.
[11:14:44 ERROR]: #!#!
``

That's the error I get.

**Comment by bensku** [comment]
Interesting. Will be fixed, but only after I have fixed new parser.

**Comment by bensku** [comment]
And as you can see, fixed it should be in next release.
